sponsoredTransactionsAuction/frontend/src/constants.ts

```diff
@@ -0,0 +1,45 @@
+import { BrowserWalletConnector, ephemeralConnectorType } from '@concordium/react-components';
+import { ContractName } from '@concordium/web-sdk';
+
+import moment from 'moment';
+
+export const VERIFIER_URL = '/api';
+
+export const NODE = 'http://node.testnet.concordium.com';
+
+export const PORT = 20000;
+
+export const REFRESH_INTERVAL = moment.duration(5, 'seconds');
```

Comment:
```suggestion
export const REFRESH_INTERVAL = moment.duration(2, 'seconds');
```

We have fast finalization now :)